wagtail/admin/static_src/wagtailadmin/scss/components/_main-nav.scss

```diff
@@ -290,11 +295,7 @@ body.explorer-open {
     }
 
     .nav-main {
-        position: absolute;
-        top: 175px; // WARNING - magic number - the height of the logo plus search box
-        bottom: 0;
         margin-bottom: 127px; // WARNING - magic number - the height of the .footer
-        width: 100%;
```

Comment:
As far as I can see none of these styles do anything with the current Flex implementation.